storybook/stories/API/cartesian/Line.stories.tsx

```diff
@@ -5,22 +5,358 @@ import { expect } from '@storybook/jest';
 import { Surface, Line, ResponsiveContainer, ComposedChart } from '../../../../src';
 import { coordinateWithNullY, coordinateData } from '../../data';
 
+// TODO: We might want to move some of these to a shared file.
+// We have a lot of duplication between the stories. Event Handlers are exposed in multiple components.
```

Comment:
oh wow, by might I think definitely 😅. Deifnitely don't want to repeat this everywhere if we can help it

Reply:
Yeah, I am not yet certain on which properties are shared where, once I figure that out the organisation of the props will have to emerge.

Reply:
Good point, I right away took out three parts - EventHandlers, Animation and Legend for reuse.